fdbrpc/tests/echo.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package fdbrpc.test;
+
+service TestEchoService {
+    rpc Echo(EchoRequest) returns (EchoResponse);
+    rpc EchoRecv10(EchoRequest) returns (stream EchoResponse);
```

Comment:
Curious, why use `10` instead of more descriptive name like `Stream`?

Reply:
Just meant that return stream of 10 messages, but makes sense to make it more description as EchoRecvStream10